e2e/test/scenarios/dashboard/reproductions/12926-editing-dashboards-doesnt-cancel-running-queries.cy.spec.js

```diff
@@ -0,0 +1,45 @@
+import {
+  editDashboard,
+  restore,
+  showDashboardCardActions,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+const PG_DB_ID = 2;
+
+const questionDetails = {
+  name: "Q1",
+  native: { query: "SELECT 1, pg_sleep(600)" },
+  database: PG_DB_ID,
+};
+
+describe("issue 12926", { tags: "@external" }, () => {
+  beforeEach(() => {
+    restore("postgres-12");
+    cy.signInAsAdmin();
+  });
+
+  describe("field source", () => {
+    it("should stop the ongoing query when removing a card from a dashboard", () => {
+      cy.createNativeQuestionAndDashboard({
+        questionDetails,
+      }).then(({ body: { dashboard_id } }) => {
+        visitDashboard(dashboard_id, { should_wait: false });
```

Comment:
`should_wait` is needed otherwise cypress will wait for the queries (that contains the sleep) to load